udemy_exercises/Sorting.java

```diff
@@ -1,25 +1,54 @@
 package udemy_exercises;
 
+import java.util.*;
+
 class Sorting {
 
+    int numArray[] = new int[10];
+
     void sortIntegers(int[] nums) {
 
+        Arrays.sort(nums);
+        System.out.println(Arrays.toString(reverse(nums)));
     }
 
     void printNums() {
-
+        System.out.println(Arrays.toString(getNums()));
     }
 
     int[] getNums() {
-        return new int[]{};
+
+        Scanner s = new Scanner(System.in);
+
+        System.out.println("Please enter 10 numbers");
+        for (int i = 0; i < 10; i++) {
+            numArray[i] = s.nextInt();
```

Comment:
could add validation so the code doesn't break if you add something other than an int